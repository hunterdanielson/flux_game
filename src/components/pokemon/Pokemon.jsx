import React from 'react';
import PropTypes from 'prop-types';
import styles from './Pokemon.css';

const Pokemon = ({ image }) => (
  <figure className={styles.Pokemon}>
    <img src={image} />
  </figure>
);

Pokemon.propTypes = {
  image: PropTypes.string.isRequired
};

export default Pokemon;
