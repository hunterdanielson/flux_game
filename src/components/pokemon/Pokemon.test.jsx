import React from 'react';
import { shallow } from 'enzyme';
import Pokemon from './Pokemon';

describe('Pokemon component', () => {
  it('renders Pokemon', () => {
    const wrapper = shallow(<Pokemon image='http://placeholder.png'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
