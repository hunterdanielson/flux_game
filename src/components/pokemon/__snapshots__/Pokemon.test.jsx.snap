// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon component renders Pokemon 1`] = `
<figure
  className="Pokemon"
>
  <img
    src="http://placeholder.png"
  />
</figure>
`;
